src/JustEat.StatsD/StatsDConfiguration.cs

```diff
@@ -51,6 +51,11 @@ public class StatsDConfiguration
         /// </summary>
         public string Prefix { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets the style for tagging. By default Disabled.
+        /// </summary>
+        public TagsStyle TagsStyle { get; set; } = TagsStyle.Disabled;
```

Comment:
As noted earlier, let's just have this be the interface for the formatter. It can have the default value of the no-op one (or even `null` and internally it would `?? new NoOp...()` at the point of assignment).

Reply:
Done.